### ISSUE TITLE
cdc2 move to cyclin -dependent term

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#1310]
check these, but I think the in vitro assay, at least for the couple I just looked at used cdk-cyclin complex, would make cdc2 MF look much better (and be more correct)

GO:0004674	protein serine/threonine kinase activity	
- [x] has substrate orc2 IDA	Vas A et al. (2001) (uses  assay in Moreno S1, Hayles J, Nurse P. which also shows cyclin dependency) PMID:11486016
- [x] has substrate srw1 IDA	Yamaguchi S et al. (2000) PMID:10921878
- [x] has substrate fkh2 IDA	Shimada M et al. (2008) PMID:18059475
- [x] has substrate cdc18  IDA	Lopez-Girona A et al. (1998) 	PMID:9436991
- [x] has substrate mde4 IMP	Choi SH et al. (2009) PMID:19523829
- [x] has substrate mde4  IMP	Choi SH et al. (2009) PMID:10921878
- [x] has substrate nsk1  IDA	Chen JS et al. (2011)  refers to In vitro kinase assays were conducted as previously described (Yoon et al., 2006),  anything  refering to this paper is cyclin-dependent assay      	PMID:22065639
- [x] has substrate byr4 IDA	Rachfall N et al. (2014) PMID:24920823

- [x] has substrate clp1  IDA	Wolfe BA et al. (2006)  (Yoon et al., 2006) PMID:16950131
- [x] has substrate xlf1 IDA	Hentges P et al. (2014)  PMID:25533340
- [x] has substrate srw1 IMP	Blanco MA et al. (2000) 	PMID:10921876

**Comment by ValWood** [comment]
will move these  - [x] down and check others later...

**Comment by ValWood** [comment]
fix in artemis
- [ ] has substrate sds23 IDA	Jang YJ et al. (1997)  PMID:9242669

**Comment by ValWood** [comment]
Looks much better
http://pombase2.aska.gen.nz/gene/SPBC11B10.09


once the final 4 are moved (if poss), and
https://github.com/pombase/website/issues/159
https://github.com/pombase/website/issues/156

the cdc2 section will  look good.
There is a *lot* of biology here....
~200 targets covering *most* processes
and all of this upstream stuff too
http://www.esyn.org//builder.php?type=Graph&term=GO:1901990&interactionType=physical&source=pombase&includeInteractors=false

(cdc2 looks to be at the centre of this network, but it is actually the "end point")

Basically if we can get this page looking meaningful, everything else should be a doddle.....

**Comment by ValWood** [comment]
Little bit of OCD there

Only this one to check/fix
has substrate cut7 IDA	Drummond DR et al. (1998) (no data in this paper has cdc2 P site?)

**Comment by ValWood** [comment]
- [ ]  only klp9 remaining

**Comment by ValWood** [comment]
all done
